### ISSUE TITLE
Error detected by Application Firewall. Please contact support if you have any questions.

### ISSUE THREAD

**Issue by fervi** [Horndev/zapread.com#72]
I suppose the bugged script blocks users (me too) from accessing the site to an IP address. You need to use TOR to use Zapread. Sometimes it's fixed, but it's not supposed to work ;)

**Comment by Horndev** [comment]
I have not tested using TOR.  I will need to investigate what may be causing this.

Do you get the same error on https://www.zapread.com (Using SSL) and http://www.zapread.com?

**Comment by fervi** [comment]
The error itself occurs on the normal Internet, then only through the TOR you can enter, because other browsers also prevent from logging in.

However, the error solves itself after a few hours, so if I will have it again, then I report.

**Comment by DocBox12** [comment]
I have the same problem. 

![zapread firewall](https://user-images.githubusercontent.com/46194605/50705683-df8ae180-105b-11e9-96a6-0c5d1adb6365.png)

**Comment by Horndev** [comment]
I am having a hard time finding the cause.  I get it if I skip the www part on Zapread.

I think it may be a web server configuration.  Still searching for a solution.

**Comment by DocBox12** [comment]
Very odd bug.

If I am running zapread.com in default Chrome profile, a error is show.

But.

If I am running zapread.com in private mode Chrome then website is a good working. 

I noticed that bug showed when I runnin zapread.com in profile browser which I use on a daily basis and I'm logged in to my profile in zapread.com

**Summary**

| Browser | Logged in to the profile in zapread | Error |
|------------|------------------------------|--------|
| Google Chrome default mode | Yes | Yes, website is not working |
| Google Chrome private mode | No | No, website is working |
| Firefox default mode | No | No, website is working |
| Google Chrome for Android default mode | No | No, website is working

I am now logged in Firefox browser and website good working. 

**Screenshots**

![zapread summary](https://user-images.githubusercontent.com/46194605/50722082-07189300-10ca-11e9-86cd-b188e4b0a3f2.png)

![screenshot_20190105-085410](https://user-images.githubusercontent.com/46194605/50722092-0f70ce00-10ca-11e9-9f0f-5f152fb8df5d.jpg)

**Comment by Horndev** [comment]
Ok, I think I solved it.  I cleared cookies and cache in chrome, and then the problem stopped:

![image](https://user-images.githubusercontent.com/908867/50728482-d273fe00-1100-11e9-876f-0cf309794a00.png)

I think this could be something from a configuration change I made on the website.  Note that now all requests go directly to SSL (https).

Anyway, I think this is the only fix.  And I think only early users from before my configuration change are affected.  I still don't understand the cause.
